scripts/distribution/macOS-package/NodeConfigurationInstallerPlugin/NodeConfigurationInstallerPlugin/MyInstallerPane.m

```diff
@@ -0,0 +1,120 @@
+//
+//  MyInstallerPane.m
+//  NodeConfigurationInstallerPlugin
+//
+//  Created by Concordium on 30/07/2021.
```

Comment:
If there is comments, they should be descriptive. 
This block doesn't state more than the obvious.

Reply:
Xcode adds these comments to all code files by default. I can remove them if you want, but they are common to see in iOS/macOS apps, including our own wallet: https://github.com/Concordium/concordium-reference-wallet-ios/tree/main/ConcordiumWallet ;)